lib/spandex_datadog/api_server.ex

```diff
@@ -346,4 +351,8 @@ defmodule SpandexDatadog.ApiServer do
   end
 
   defp deep_remove_nils(term), do: term
+
+  defp term_to_string(term) when is_binary(term), do: term
+  defp term_to_string(term) when is_atom(term), do: term
```

Comment:
This returns an atom, not a string. Are we doing that just because atoms are encoded as strings?

Reply:
Also the test you added doesn't test an atom value, so I can't determine if this is correct behavior from your test.

Reply:
I tested it locally and it works, I'll merge this and push my test changes.